File: python/easy/destination_city.py
# 1436. Destination City
# https://leetcode.com/problems/destination-city/

class Solution:
    def destCity(self, paths: List[List[str]]) -> str:
        ans = []
        
        for i in paths:
            ans.append(i[0])
        
        for i in paths:
            if i[1] not in ans:
                return i[1]

File: javascript/easy/destination_city.js
// 1436. Destination City
// https://leetcode.com/problems/destination-city/

var destCity = function(paths) {
    var ans = [];
    
    for (var i = 0; i < paths.length; i++) {
        ans.push(paths[i][0]);
    }
    
    for (var i = 0; i < paths.length; i++) {
        if ( !(ans.includes(paths[i][1])) ) {
            return paths[i][1];
        }
    }
};

console.log(destCity([["London","New York"],["New York","Lima"],["Lima","Sao Paulo"]]))